internal/backend/db/dbgorm/deployments.go

```diff
@@ -201,3 +202,29 @@ func (db *gormdb) ListDeployments(ctx context.Context, filter sdkservices.ListDe
 		return kittehs.TransformError(ds, scheme.ParseDeployment)
 	}
 }
+
+func (db *gormdb) DeploymentHasActiveSessions(ctx context.Context, id sdktypes.DeploymentID) (bool, error) {
+	r, err := db.ListSessions(ctx, sdkservices.ListSessionsFilter{
+		DeploymentID: id,
+		StateType:    sdktypes.SessionStateTypeRunning,
```

Comment:
not for this PR of coz, but maybe ListSessions could receive the several states? e.g `Running, Created`? 
or bit mask: `Running | Created`?

Reply:
maybe in the future.